source/use-cases/home-resolver.rst

```diff
@@ -1,128 +1,200 @@
 
 The example config is found at:
-::
+
+.. code-block:: bash
+
 	/etc/unbound/unbound.conf
 
-if you open this for the first time it looks very empty. It is still usable for one machine, as this is how all the Unbound defaults are configured. It's not, however, enough for our purposes so we will add the minimal configuration options.
+If you open this for the first time it looks very empty. It is still usable for one machine, as this is how the Unbound defaults are configured. It's not, however, enough for our purposes, so we will add the minimal configuration options needed.
+
+The options that we add to the current config file to make it a "minimal usable config" are as follows. Note that the IPv6 options are commented out, but we recommend to uncomment them if your router and network supports it.
+
+.. code-block:: bash
 
-The options that we add to the current config file to make it a "minimal usable config" are:
-::
 	server:
-            # the interface that is used to connect to the network, this means on this machine
-            interface: 0.0.0.0
-            interface: ::0
-            # addresses from the IP range that are allowed to connect to the resolver
-            access-control: 10.0.0.0/8 allow
-            access-control: 2001:DB8::/64 allow
+			# location of the trust anchor file that enables DNSSEC
+			auto-trust-anchor-file: "/var/lib/unbound/root.key"
+			# the interface that is used to connect to the network, this means on this machine
+			interface: 0.0.0.0
+			# interface: ::0
+			# addresses from the IP range that are allowed to connect to the resolver
+			access-control: 10.0.0.0/8 allow
+			# access-control: 2001:DB8.. code-block:: bash/64 allow
+	remote-control:
+			control-enable: yes
+
+The access-control is currently configured to listen to any address on the machine, and only allow queries from the ``10.0.0.0/8`` `IP subnet <https://www.ripe.net/about-us/press-centre/understanding-ip-addressing>`_ range. Note that the IP addresses we chose (``10.0.0.1`` and ``10.0.0.2``) fall within the ``10.0.0.0/8`` range.
 
-The access-control is currently configured to listen to any address on the machine, and only allow queries from the 10.0.0.0/8 IP range.
+To prepare our config we are going to modify the existing config in :file:`/etc/unbound/unbound.conf`.
+If you open the file we see that there is already an “include” in there. This include enables us to do `DNSSEC <https://en.wikipedia.org/wiki/Domain_Name_System_Security_Extensions>`_, which allows Unbound to verify the source of the answers that it receives, which we want to keep in. If you don't have the files that the include links to, they can be created using the :command:`unbound-anchor` command.
```

Comment:
I *think* the include refers to file unbound.conf.d/root-auto-trust-anchor-file.conf.  unbound-anchor command doesn't create this file, but generates /var/lib/unbound/root.key.